src/main/java/racingcar/domain/Rounds.java

```diff
@@ -0,0 +1,35 @@
+package racingcar.domain;
+
+import racingcar.dto.RoundResult;
+import racingcar.dto.RoundResults;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Rounds {
+    private final List<Round> rounds;
+
+    public Rounds(List<Round> rounds) {
+        this.rounds = rounds;
+    }
+
+    public RoundResults play() {
+        List<RoundResult> roundResults = new ArrayList<>();
+
+        for (Round round : rounds) {
+            roundResults.add(round.play());
+        }
+
+        return new RoundResults(roundResults);
```

Comment:
stream을 적극 활용하셔서 메서드 레벨에서 FP를 구현해보셔도 좋을 것 같습니다.

[Java8 Stream Examples](https://ahndding.tistory.com/23)
```suggestion
        return new RoundResults(rounds.stream()
                .map(Round::play)
                .collect(Collectors.toList()));
```